src/WCSimConstructMaterials.cc

```diff
@@ -1572,8 +1714,10 @@ void WCSimDetectorConstruction::ConstructMaterials()
 
 
   WlsPlasticMPT->AddProperty("RINDEX",WLSProps->GetPhotonEnergy(),WLSProps->GetRIndex(),WLSProps->GetNumEntries());
+#if 1 // turn to 0 to make plate non-reemitting, purely passive
```

Comment:
the default if 1, so not passive. If you turn it to 0, it becomes passive. This is used to simulate the aged detector (after 20 years of running), under the pessimistic assumption that the plates age badly and turn completely passive.
I had to do it via #if and #endif because the config file does not work - the value you put in the config file is then overwritten again at run time with the default value. It's always bothered me because of course it would be better to control it via mac file and not recompile. But I never could make it work for some of the options (some work, some don't)

Reply:
wanna do something similar to `WCSIM_SAVE_PHOTON_HISTORY_FLAG`?
https://github.com/WCSim/WCSim/blob/8cc7f87ae1f9f8c96775cd0c2204c80e762fc9dc/CMakeLists.txt#L145

Reply:
Closing this, as it is quite a niche thing to do (and the default is sensible). If HK FD or IWCD OD groups think implementing this via a messenger or cmake command, then please implement it